### PR TITLE
Remove logging values in p2p lib

### PR DESCRIPTION
**Pull request by pjenys0**
Don't log values of data when read/write to db

### CODE REVIEW

File: p2p/kademlia/store/db/db_test.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import (
+	"context"
+	"crypto/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDB(t *testing.T) {
```
Comment (leonidcode20) [inline p2p/kademlia/store/db/db_test.go:12]: should be `testing.B`

File: p2p/kademlia/store/db/db_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	ctx, cancel := context.WithTimeout(context.Background(), 100*time.Second)
+	defer cancel()
+	store, err := NewStore(ctx, "/tmp/testDb")
+	assert.Nil(t, err)
+
+	for i := 0; i <= 2000; i++ {
+		key := make([]byte, 32)
+		rand.Read(key)
+		writeData := make([]byte, 50*1024)
+		rand.Read(writeData)
```
Comment (leonidcode20) [inline p2p/kademlia/store/db/db_test.go:23]: Should bench Read/Write separately.

File: p2p/kademlia/store/db/db_test.go (hunk 1 above)
Comment (pjenys0) [reply p2p/kademlia/store/db/db_test.go:12]: Nop, this include logic testing.

File: p2p/kademlia/store/db/db_test.go (hunk 2 above)
Comment (pjenys0) [reply p2p/kademlia/store/db/db_test.go:23]: This is not bench test. just try to do thousand write/read, and ensure no timeout happen.

**Review by leonidcode20 — APPROVED** [review]

**Review by danileng20 — APPROVED** [review]
LGTM!

**Pull request merged**